hm-post-repeat.php

```diff
@@ -440,3 +449,142 @@ function get_repeating_post( $post_id ) {
 	return $original_post_id;
 
 }
+
+/**
+ * Adds admin table view link per available repeat type.
+ * So that only all posts of specific repeat type are displayed.
+ *
+ * Added at the end of link list for All | Mine | Published | Scheduled | Drafts
+ *
+ * @param array $views An array of available list table views.
+ *
+ * @return array Available list of table views with custom added views per repeat type.
+ */
+function admin_table_views_links( $views ) {
+
+	// Add status link for each repeat type.
+	foreach ( get_available_repeat_types() as $repeat_type => $repeat_desc ) {
+
+		$url_args = array(
+			'post_type'      => get_current_screen()->post_type,
+			'hm-post-repeat' => $repeat_type,
+		);
+
+		// Custom WP_Query to get posts count of repeat type.
+		$repeat_type_query = new \WP_Query( get_repeat_type_query_params( $repeat_type ) );
+
+		$link_label = sprintf(
+				'%s <span class="count">(%s)</span>',
+			$repeat_desc,
```

Comment:
I _think_ this needs escaping as is being output in html, or are these links escaped later by core?

Reply:
Hm, doesn't look like it - I'll add.